src/main/java/com/manardenza/Main.java

```diff
@@ -1,5 +1,6 @@
 package com.manardenza;
 
+import com.manardenza.utils.DatabaseSeeder;
 import org.slf4j.LoggerFactory;
 
 public class Main {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)